modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx

```diff
@@ -23,7 +23,7 @@ import React, {useEffect, useState} from 'react';
 
 import {IFDSViewSectionProps} from '../../../FDSView';
 import {FDSViewType} from '../../../FDSViews';
-import {getFields} from '../../../api';
+import {getFields, isSortable} from '../../../api';
```

Comment:
Is `isSortable` just imported in this one file? If yes, we can put it here and not export it.